compiler/onecc-docker/tests/prepare_test_materials.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# See https://github.com/Samsung/ONE/issues/4155 for information
+
+SCRIPT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+pushd $SCRIPT_PATH > /dev/null
+
+if [[ ! -s "bcq.pb" ]]; then
+    rm -rf bcq.pb.zip
+    wget https://github.com/Samsung/ONE/files/5153842/bcq.pb.zip
+    unzip bcq.pb.zip
+    # https://github.com/Samsung/ONE/issues/4155#issuecomment-689324597
+fi
```

Comment:
@seanshpark 
What about this model?

Reply:
I would say small tflite source would be better.
Anyway why did you choose this one in particular?

Reply:
It's just because I thought this model was pretty small.

I'm sorry, but I couldn't find the `tflite` model except for `inception_v3`. I'll ask where I can get the tflite

Reply:
Sorry for incorrect direction... -_-;
>    unzip bcq.pb.zip
    # https://github.com/Samsung/ONE/issues/4155#issuecomment-689324597

When you open above link, you'll find some explanation of the model and as you can see, the attachment model.

Likewise, you can add a comment, a very short simple single sentence one will do, in #9712 issue and attach .zip model found in `build/compiler/common-artifacts`.
I think any well known operator like Conv2D or Add will do.